kubeface/broadcast.py

```diff
@@ -0,0 +1,21 @@
+import hashlib
+import logging
+
+class Broadcast(object):
```

Comment:
Basic idea looks good! Couple of suggestions:
 * Use getstate / setstate to write out the data while pickling `Broadcast` for the first time instead of in `__init__`. This way the broadcast is only done if it is actually used. With this approach `value` would then be an attribute not a method.
 * Need to pickle the `data`, not write it out as a string
* Need to use `storage` functions instead of filesystem so we support buckets
* Should probably add something to `naming` that defines how we name broadcasts. Should probably include the job name and a random string, instead of the hash of the value. Using the hash of the value is nice but would require serializing the data twice as implemented here.
* We don't want to assume that we have enough memory to hold the serialized value in memory. It should always be written to and read from file handles, never held as a string in memory